src/tests/fixtures/users.fixture.js

```diff
@@ -63,6 +63,7 @@ export const loggedInToken = JwtService.generateToken({
 });
 
 export const createUsers = async () => {
+  await Users.destroy({ where: {} }).then(() => {});
```

Comment:
There is no need for the `then` callback. You can remove it

Reply:
Ok